amplify/team-provider-info.json

```diff
@@ -15,25 +15,29 @@
       "function": {
         "expressServer": {
           "deploymentBucketName": "amplify-stockapp-dev-151619-deployment",
-          "s3Key": "amplify-builds/expressServer-746764634d6933773954-build.zip"
+          "s3Key": "amplify-builds/expressServer-753174514c376f647362-build.zip"
```

Comment:
What are these changes? Other than being unsure of these everything looks good to me